### DISCUSSION TITLE
Is it possible to add a line feed to 'text'

### DISCUSSION THREAD

**Discussion by JayIsPainting** [bluesky-social/atproto#1866] [How to do it?]
I'd like to programmatically assemble the text for a post and I am wondering if there's any way to do it. Seems like anything I try posts as literal characters. A security thing no doubt.

**Comment by snarfed** [comment]
Are you looking for facets? https://atproto.com/blog/create-post#mentions-and-links

**Reply by JayIsPainting** [reply]
Maybe? I'm using them to parse and display clickable URLs but I'm not sure how they apply to adding a crlf.

Is it via ByteSlice?

**Reply by DavidBuchanan314** [reply]
There's no special trick to it, you only need a line feed (\n) but I'm sure crlf doesn't hurt either.
